deployment/1.7/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Jakarta Deployment Specification 1.7"
+date: 2019-07-30T10:00:00+0100
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Deployment defines standard APIs that will enable any deployment tool that uses the
+deployment APIs to deploy any assembled application onto a Jakarta EE compatible platform.
+
+* [Jakarta Deployment 1.7 Specification Document](./deployment-spec-1.7.pdf) (PDF)
+* [Jakarta Deployment 1.7 Specification Document](./deployment-spec-1.7.html) (HTML)
+* [Jakarta Deployment 1.7 Javadoc](./apidocs)
+* Maven coordinates
```

Comment:
Add link to platform TCK

Reply:
Remove timestamp when updating _index.md

Reply:
Fixed